code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -14,12 +14,32 @@
 	var/turf_type = null
 	var/mineralType = null
 	novariants = TRUE
+	var/human_maxHealth = 100
 
 /obj/item/stack/tile/Initialize(mapload, amount)
 	. = ..()
 	pixel_x = rand(-3, 3)
 	pixel_y = rand(-3, 3) //randomize a little
 
+/obj/item/stack/tile/examine(mob/user)
+	..()
```

Comment:
Preserve the return and add the extra text to it, instead of adding a second `to_chat()`